src/internal/connector/graph_connector.go

```diff
@@ -191,6 +198,79 @@ func (gc *GraphConnector) GetUsersIds() []string {
 	return buildFromMap(false, gc.Users)
 }
 
+// setTenantSites queries the M365 to identify the sites in the
+// workspace. The sitets field is updated during this method
+// iff the return value is nil
+func (gc *GraphConnector) setTenantSites(ctx context.Context) error {
+	// TODO
+	gc.Sites = map[string]string{}
```

Comment:
Why would `Sites` become a visible field for GC?

Reply:
Sites are the acting Resource Owner for SharePoint application handling.  We don't catalog data by users, so there's not much point in retrieving them.  I'd say a more generic means of handling this would be to swap both Users and Sites for a `ResourceOwner` map and pivot on the type as needed, but I know ark also uses this to retrieve users, so I don't want to futz with that at this moment.